README.rst

```diff
@@ -31,10 +31,9 @@ Apache Solr is based on Lucene and is *the* enterprise open source search engine
 It powers the search of sites like Twitter,
 the Apple and iTunes Stores, Wikipedia, Netflix and many more.
 
-Solr does not only scale to any level of content,
-but provides rich search functionality,
-like faceting, geospatial search, suggestions, spelling corrections, indexing of binary formats and a whole variety of powerful tools to configure custom search solutions.
-It has integrated clustering and load-balancing to provide a high level of robustness.
+For the wide range of functionality provided by Solr, see `Solr features`_.
```

Comment:
@gforcada linking to this page is a good idea. Though, I would somehow like to still list the main Solr features for convenience.